client/question/components/LearnerViewDirectiveSpec.js

```diff
@@ -421,4 +423,18 @@ describe('LearnerViewDirective', function() {
       expect(EventHandlerService.createTaskCompleteEvent).toHaveBeenCalled();
     });
   });
+
+  describe('supportedLanguageLabels', function() {
+    it('should provide a descriptive label for each language', function() {
+      expect($scope.supportedLanguageLabels[LANGUAGE]).toEqual('Python 2.7');
+
+      expect($scope.supportedLanguageCount).toEqual(
+        ALL_SUPPORTED_LANGUAGES.length);
+
+      for (var i = ALL_SUPPORTED_LANGUAGES.length - 1; i >= 0; i--) {
+        var $label = $scope.supportedLanguageLabels[ALL_SUPPORTED_LANGUAGES[i]];
```

Comment:
Why the "$" prefix? Should just be `label`.

Reply:
Removed $ label. PHP programming tic.